### DISCUSSION TITLE
Using w.config object to retrieve custom data added to a series

### DISCUSSION THREAD

**Discussion by john-angel** [apexcharts/apexcharts.js#2648] [Q&A]
Hi, I've seen the w.globals object can be used to display in a tooltip the custom data added to the series object, however I would like to confirm if there's a risk by using the w.config object. I made a test with a column bar and so far it's working fine.

```javascript
const barSeries = [
        {
            name: 'Series',
            data: [{
                       x:0,
                       y:10,
                       licensed:5
            },
            {...}
           ]
        }
]
```

In the tooltip option I'm getting the custom data from w.config:

```javascript
custom:function ({series, seriesIndex, dataPointIndex, w})
    const licensed = w.config.series[0].data[dataPointIndex].licensed;
    return (...)
}
```
Please let me know if this is ok or I should use w.globals.

Thank you!

**Comment by alexprey** [comment]
I'm try to do few days ago and that not works perfectly. Sometimes extra data in series gone. Works fine with 1-2 series, with more series that works non predictable. As a workaround I'm store data in global scope

**Comment by goodkiller** [comment]
Extra data is very handy, please add this possibility!
